installing-a-promise/README.md

```diff
@@ -187,15 +189,6 @@ To recap the steps you took:
 
 This is only the beginning of working with Promises. Next you will deploy three different Promises to provide a complete solution for an application team.
 
-## <a name="teardown"></a>Tearing it all down
-The next section in this tutorial requires a clean Kratix installation. Before heading to it, please clean up your environment by running:
-
-```bash
-kind delete clusters platform worker
-```
-<br />
-
-
```

Comment:
Step 3 involves running 
```
cd /path/to/kratix
./scripts/quick-start.sh --recreate
```

which includes deleting the existing clusters for you